src/foam/dao/ORBitalDAO.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao',
+  name: 'ORBitalDAO',
+  documentation : 'DAO class used to delegate a method execution request to the specified object (specified by an ORBRequest object).',
+  extends: 'foam.dao.ProxyDAO',
+  flags: ['java'],
+
+  javaImports: [
+    'foam.core.FObject',
+    'foam.core.ClassInfo',
+    'foam.core.MethodInfo',
+    'foam.core.X',
+    'foam.nanos.logger.Logger'
+  ],
+
+  properties: [
+
+  ],
+
+  methods: [
+    {
+      name: 'cmd_',
+      type: 'Object',
+
+      args: [
+        { name: 'x' , type: 'foam.core.X'}, { name: 'obj' , type: 'FObject' }
+      ],
+
+      javaCode: `
+        ORBRequest request = null;
```

Comment:
Since the ORBitalDAO is a Proxy, rather than the try/catch, test if the obj is an instance of ORBRequest, if it is then process it, else pass it on to the delegate.

Reply:
OK